Boot2Docker.iss

```diff
@@ -17,7 +17,7 @@
 ; https://www.virtualbox.org/wiki/Downloads
 ; Then, run "VirtualBox-x.x.x-xxx-Win.exe --extract --path ."
 #define virtualBoxCommon ".\VirtualBox\common.cab"
-#define virtualBoxMsi ".\VirtualBox\VirtualBox-4.3.20-r96997-MultiArch_amd64.msi"
+#define virtualBoxMsi ".\VirtualBox\VirtualBox-4.3.26-r98988-MultiArch_amd64.msi"
```

Comment:
Do you think it'd be worth renaming this file in `build.sh` so that we can keep all our version numbers in one central place? (and thus simplify version bumps)